src/Log/LogTrait.php

```diff
@@ -27,13 +27,13 @@ trait LogTrait
      * Convenience method to write a message to Log. See Log::write()
      * for more information on writing to logs.
      *
-     * @param mixed $msg Log message.
+     * @param string $message Log message.
      * @param int|string $level Error level.
      * @param string|array $context Additional log data relevant to this message.
      * @return bool Success of log write.
      */
-    public function log($msg, $level = LogLevel::ERROR, $context = []): bool
+    public function log(string $message, $level = LogLevel::ERROR, $context = []): bool
```

Comment:
Will this break public API of e.g. https://github.com/dereuromark/CakePHP-DatabaseLog/blob/cake4/src/Log/Engine/DatabaseLog.php#L49 in a BC breaking way?
Then we dont need to add the typehint here either, the docblock fix should suffice.

Reply:
No it won't. Log engines use neither `Cake\Log\Log.php` or `Cake\Log\LogTrait`. You engine extends `Cake\Log\Engine\BaseLog`.